### PR TITLE
4.0 python example connecting to a service 2 4

### PR DESCRIPTION
**Pull request by martin-neotech**
Added python tab to the section 2.4

### CODE REVIEW

File: asciidoc/client-applications.adoc [hunk 1]
```diff
@@ -728,6 +728,127 @@ To connect to a service without routing, you can replace `neo4j` with `bolt`.
...
+
+|Neo4j Aura
+|Secured with full certificate
+|
+[source, python]
+----
+GraphDatabase.driver("neo4j+s://graph.example.com:7687", auth)
+----
+
+For previous Python Driver versions before 4.0.0 you will need this snippet instead:
```
Comment (AlexicaWright) [inline asciidoc/client-applications.adoc]:
```suggestion
For Python Driver versions before 4.0.0 you need this snippet instead:
```

File: asciidoc/client-applications.adoc [hunk 2]
```diff
@@ -728,6 +728,127 @@ To connect to a service without routing, you can replace `neo4j` with `bolt`.
...
+
+|Neo4j 4.x
+|Secured with full certificate
+|
+[source, python]
+----
+GraphDatabase.driver("neo4j+s://graph.example.com:7687", auth)
+----
+
+For previous Python Driver versions before 4.0.0 you will need this snippet instead:
```
Comment (AlexicaWright) [inline asciidoc/client-applications.adoc]:
```suggestion
For Python Driver versions before 4.0.0 you need this snippet instead:
```

File: asciidoc/client-applications.adoc [hunk 3]
```diff
@@ -728,6 +728,127 @@ To connect to a service without routing, you can replace `neo4j` with `bolt`.
...
+
+|Neo4j 4.x
+|Secured with self-signed certificate
+|
+[source, python]
+----
+neo4j.driver("neo4j+ssc://graph.example.com:7687", auth)
+----
+
+For previous Python Driver versions before 4.0.0 you will need this snippet instead:
```
Comment (AlexicaWright) [inline asciidoc/client-applications.adoc]:
```suggestion
For Python Driver versions before 4.0.0 you need this snippet instead:
```

File: asciidoc/client-applications.adoc [hunk 4]
```diff
@@ -728,6 +728,127 @@ To connect to a service without routing, you can replace `neo4j` with `bolt`.
...
+
+|Neo4j 3.x
+|Secured with full certificate
+|
+[source, python]
+----
+neo4j.driver("neo4j+s://graph.example.com:7687", auth)
+----
+
+For previous Python Driver versions before 4.0.0 you will need this snippet instead:
```
Comment (AlexicaWright) [inline asciidoc/client-applications.adoc]:
```suggestion
For Python Driver versions before 4.0.0 you need this snippet instead:
```

File: asciidoc/client-applications.adoc [hunk 5]
```diff
@@ -728,6 +728,127 @@ To connect to a service without routing, you can replace `neo4j` with `bolt`.
...
+
+|Neo4j 3.x
+|Secured with self-signed certificate
+|
+[source, python]
+----
+neo4j.driver("neo4j+ssc://graph.example.com:7687", auth)
+----
+
+For previous Python Driver versions before 4.0.0 you will need this snippet instead:
```
Comment (AlexicaWright) [inline asciidoc/client-applications.adoc]:
```suggestion
For Python Driver versions before 4.0.0 you need this snippet instead:
```

**Review by AlexicaWright — COMMENTED** [review]
Suggestions for better readability.

**Review by davidoliverSP2 — COMMENTED** [review]
Thanks @martin-neotech - I agree with the feedback from @AlexicaWright 

Is there a plan for adding the rest of the 4.0 Python driver examples? On the [introduction page](https://neo4j.com/docs/driver-manual/4.0/get-started/#driver-get-started-about) we talk about Python not yet being available - adding in this one example may cause confusion. When Python is available we'll need to update all examples, this admonition, and the [Drivers section of the Migration Guide](https://neo4j.com/docs/migration-guide/4.0/upgrade-driver/#upgrade-driver-releases).

**Comment by martin-neotech** [comment]
@davidoliverSP2 Yes, I am updating all the python docs now for driver 4.0, all the examples have been updated soon and will be available with the next release.

**Pull request merged**